app/src/types/ChefsFormConfig.ts

```diff
@@ -0,0 +1,10 @@
+export type ChefsFormConfig = {
+  form1: ChefsFormConfigData;
+  form2: ChefsFormConfigData;
+};
+
+export type ChefsFormConfigData = {
+  name: string;
+  formId: string;
+  formApiKey: string;
```

Comment:
Could be 'id' and 'apiKey' from earlier style comment if we wanted to be more terse.